plugins/test/unit/plugins/importers/yum/test_upload.py

```diff
@@ -376,14 +387,57 @@ def test_handle_for_category(self):
         saved_unit = mock_conduit.save_unit.call_args[0][0]
         self.assertEqual(inited_unit, saved_unit)
 
+    @mock.patch('pulp_rpm.plugins.importers.yum.upload._get_file_units')
+    def test_handle_for_comps(self, mock_file_units):
+        # Setup
+        self.sample_comps_filename = os.path.join(DATA_DIR, 'simple_repo_comps', XML_FILENAME)
+        unit_key = {}
+        metadata = {}
+        config = PluginCallConfiguration({}, {})
+        mock_repo = mock.MagicMock()
+        mock_repo.id = 'some_id'
+        mock_conduit = mock.MagicMock()
+
+        # Test
+        upload._handle_group_category(mock_repo, models.PackageCategory.TYPE, unit_key, metadata,
+                                      self.sample_comps_filename, mock_conduit, config)
+
+        # Verify
+        self.assertEqual(mock_file_units.call_count, 3)
```

Comment:
I think we should have some more assertions here than how many times _get_file_units was called. We should check the three calls to assert that they were passed the correct arguments.